src/PrestaShopBundle/Form/Admin/Product/ProductInformation.php

```diff
@@ -52,12 +52,17 @@
 use Symfony\Component\Form\FormEvents;
 use Symfony\Component\Translation\TranslatorInterface;
 use Symfony\Component\Validator\Constraints as Assert;
+use Symfony\Component\Validator\Constraints\Length;
 
 /**
  * This form class is responsible to generate the basic product information form.
  */
 class ProductInformation extends CommonAbstractType
 {
+    //Min and Max Product Name length
+    const MAX_PRODUCT_NAME = 128;
+    const MIN_PRODUCT_NAME = 3;
```

Comment:
Just to be picky, we try to be explicit 🙂
```suggestion
    public const MAX_PRODUCT_NAME = 128;
    public const MIN_PRODUCT_NAME = 3;
```

Reply:
We even tried to implement a custom PHPStan rule to enforce it 😄 
https://github.com/PrestaShop/phpstan-prestashop/blob/main/src/Rules/ClassConstantsMustHaveVisibilityRule.php